src/infer/inferOs.js

```diff
@@ -11,7 +11,7 @@ function inferPlatform() {
 
 function inferArch() {
   const arch = os.arch();
-  if (arch !== 'ia32' && arch !== 'x64') {
+  if (arch !== 'ia32' && arch !== 'x64' && arch !== 'arm') {
```

Comment:
@ronjouch Mostly just on desktop, but I can see this being super useful for projects on devices like the pi